strategy-offline-fallback/controlled.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Offline fallback: controlled page - ServiceWorker Cookbook</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+   body {
+     text-align: center;
+   }
+  </style>
+</head>
+<body>
+  <h1>Offline fallback</h1>
+  <img src="" alt="missing asset" />
+  <p>This image request points to an unavailable resource but since the page
```

Comment:
Add some commas so we can breathe while reading this!

[...] resource**,** [...] fallback**,**